### PR TITLE
chore: update README

### PR DESCRIPTION
**Pull request by olizilla**
Fix headings and ordering. Pull in best bits from the existing README.

License: MIT
Signed-off-by: Oli Evans <oli@tableflip.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,68 +1,81 @@
...
 [![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg)](https://protocol.ai/) [![](https://img.shields.io/badge/project-IPFS-blue.svg)](http://ipfs.io/) [![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg)](http://webchat.freenode.net/?channels=%23ipfs) [![Build Status](https://travis-ci.org/ipfs-shipyard/ipfs-webui.svg?branch=revamp)](https://travis-ci.org/ipfs-shipyard/ipfs-webui) [![dependencies Status](https://david-dm.org/ipfs-shipyard/ipfs-webui/revamp/status.svg)](https://david-dm.org/ipfs-shipyard/ipfs-webui/revamp)
 
-## Background
-
-### This is pre-release software.
-
-The current IPFS webui is here: https://github.com/ipfs-shipyard/ipfs-webui
 
-This repo is part of the IPFS GUI redesign project, described here: https://github.com/ipfs-shipyard/pm-ipfs-gui
+The IPFS WebUI is a **work-in-progress**. Help us make it better! We use the issues on this repo to track the work and it's part of the wider [IPFS GUI project](https://github.com/ipfs/ipfs-gui)
```
Comment (fsdiogo) [inline README.md]: Missing `.` at the end.

File: README.md [hunk 2]
```diff
@@ -1,68 +1,81 @@
...
 
-The app accesses a local IPFS daemon via [`window.ipfs-fallback`](https://github.com/tableflip/window.ipfs-fallback). It will use the `window.ipfs` api provided by the [IPFS Companion](https://github.com/ipfs-shipyard/ipfs-companion) web-extension where available, and fallback to using [js-ipfs-api](https://github.com/ipfs/js-ipfs-api)
+The app uses the IPFS http api to get data from the ipfs node. It will use the `window.ipfs` api provided by the [IPFS Companion](https://github.com/ipfs-shipyard/ipfs-companion) web-extension where available, and fallback to using [js-ipfs-api](https://github.com/ipfs/js-ipfs-api) where not.
```
Comment (fsdiogo) [inline README.md]: `HTTP API`, all uppercase.

File: README.md [hunk 3]
```diff
@@ -1,68 +1,81 @@
...
 
 The app is built with [`create-react-app`](https://github.com/facebook/create-react-app). Please read the [docs](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#table-of-contents).
 
 ## Install
 
-With `node` > 8.9 and `npm` @ 6+ installed, run
+With `node` >= 8.9 and `npm` >= 6.0 installed, run
 
 ```js
```
Comment (fsdiogo) [inline README.md]: It should be `sh` and have the `>`.

```sh
> npm install
```

File: README.md [hunk 4]
```diff
@@ -1,68 +1,81 @@
...
+
+## Build
+
+To create an optimized static build of the app, output to the `build` directory:
 
 ```sh
-# Run bundle
-npm run analyze
+# Build out the html, css & jss to ./build
+npm run build
```
Comment (fsdiogo) [inline README.md]: Add the `>`.

File: README.md [hunk 5]
```diff
@@ -75,15 +88,7 @@ npm test
 
 The uses Jest to run the isolated unit tests. Unit test files are located next to the component they test and have the same file name, but with the extension `.test.js`
```
Comment (fsdiogo) [inline README.md]: The `WebUI` uses Jest (...)

File: README.md [hunk 6]
```diff
@@ -75,15 +88,7 @@ npm test
...
-The following command will perform [`standard`](https://standardjs.com/) linting on the code:
-
-```sh
-npm run lint
-```
-
-### End-to-end tests
+## End-to-end tests
 
 The end-to-end tests (e2e) test the full app in a headless Chromium browser. They require an http server be running to serve the app.
```
Comment (fsdiogo) [inline README.md]: They require an http server `to` be running to serve the app.

**Comment by hacdias** [comment]
@olizilla was just passing by and made the changes according to @fsdiogo's comments.

**Pull request merged**